### PR TITLE
Monitoring: Move monitoring.tsx to monitoring/alerting.tsx

### PR DESCRIPTION
**Pull request by kyoto**
This moves the last of the admin perspective monitoring code under the
components/monitoring/ directory.

### CODE REVIEW

**Comment by kyoto** [comment]
Rebased

File: frontend/public/components/app-contents.tsx [hunk 1]
```diff
@@ -438,7 +438,7 @@ const AppContents_: React.FC<AppContentsProps> = ({ activePerspective }) => {
               path="/monitoring/alerts"
               exact
               loader={() =>
-                import('./monitoring' /* webpackChunkName: "monitoring" */).then(
+                import('./monitoring/alerting' /* webpackChunkName: "monitoring" */).then(
```
Comment (spadgett) [inline frontend/public/components/app-contents.tsx:441]: Should we change the webpack chunk name too?

Comment (kyoto) [reply frontend/public/components/app-contents.tsx]: 👍 Changed to `alerting`

**Comment by kyoto** [comment]
Rebased again

**Comment by kyoto** [comment]
Fixed rebase

**Pull request merged**